games/gamble/gamble.py

```diff
@@ -28,6 +29,17 @@ async def roll(self, ctx, max_roll: int = 6):
     async def flip(self, ctx):
         await ctx.send(random.choice([config["heads"], config["tails"]]))
 
+    @commands.command(name="slots", brief="Try your hand at the slots and get rich.")
+    async def slots(self, ctx, *args):
+        if (len(args) == 0):
+            bet = None
+        elif (len(args) != 1):
```

Comment:
if we're only allowing one arg why not just have a named param

Reply:
i think i did it this way just for that error msg. can prolly remove it.